schema-admin.graphql

```diff
@@ -264,6 +264,11 @@ type Query {
   getCollectionPartnerAssociationForCollection(
     externalId: String!
   ): CollectionPartnerAssociation
+
+  """
+  Retrieves all available Labels
+  """
+  labels: [Label!]!
```

Comment:
In line with conventions established for other queries for support entities like this one (e.g. curation categories), I'm guaranteeing returning something to the user - this makes it easier to work with query results on the frontend.